vm/systemSmartContracts/staking.go

```diff
@@ -530,6 +636,10 @@ func (r *stakingSC) unBond(args *vmcommon.ContractCallInput) vmcommon.ReturnCode
 		log.Error("unBond is not possible for jailed nodes")
 		return vmcommon.UserError
 	}
+	if !r.canUnBond() {
```

Comment:
if the unstake transaction was processed, there should be no problem to unbond because the computer could be already offline

Reply:
yes, it could. But it is still a small protection that he may leave it online until it is needed.
otherwise there is the risk for him to have his stake bled.

Reply:
this defeats the purpose of having a minimum preefined period between the unstake and unbond. The unstake transaction should not be processed if there are too few validators. After the unstake, the validator is no longer a validator and cannot harm the network, but could still be punished. If he works til the last day before unbonding, there is no time to punish him any more.